src/odemis/acq/stitching/_tiledacq.py

```diff
@@ -222,13 +224,15 @@ def _moveToTile(self, idx, prev_idx, tile_size):
             m["y"] = self._starting_pos["y"] - idx[1] * tile_size[1] * overlap
 
         logging.debug("Moving to tile %s at %s m", idx, m)
-        f = self._stage.moveAbs(m)
+        self._future.running_subf = self._stage.moveAbs(m)
         try:
             speed = min(self._stage.speed.value.values()) if model.hasVA(self._stage, "speed") else 10e-6
             # add 1 to make sure it doesn't time out in case of a very small move
             t = math.hypot(abs(idx_change[0]) * tile_size[0] * overlap,
                            abs(idx_change[1]) * tile_size[1] * overlap) / speed + 1
-            f.result(t)
+            self._future.running_subf.result(t)
+            # TODO: instead of sleeping, handle vibration effects
+            time.sleep(1)
```

Comment:
Well, it should be a FIXME instead of a TODO, because it's really bad.
I think that it should be either handled in the stage driver (eg, add the sleep() in smaract.5DoF), or the stage configuration (ie, change some parameters in the microscope file or the controller). First, more testing is needed to understand what are these vibrations exactly, and how can they be handled most properly.